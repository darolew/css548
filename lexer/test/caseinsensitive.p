and
AND
AnD
array
ARRAY
ArraY
begin
BEGIN
BegiN
case
CASE
CasE
const
CONST
ConsT
div
DIV
DiV
do
DO
Do
downto
DOWNTO
DowntO
else
ELSE
ElsE
end
END
EnD
for
FOR
FoR
function
FUNCTION
FunctioN
if
IF
If
in
IN
In
mod
MOD
Mod
nil
NIL
NiL
not
NOT
NoT
of
OF
Of
or
OR
Or
procedure
PROCEDURE
ProcedurE
program
PROGRAM
PrograM
record
RECORD
RecorD
repeat
REPEAT
RepeaT
set
SET
SeT
then
THEN
TheN
to
TO
To
type
TYPE
TypE
until
UNTIL
UntiL
var
VAR
VaR
while
WHILE
WhilE

false
FALSE
FalsE
true
TRUE
TruE
new
NEW
NeW
dispose
DISPOSE
DisposE
read
READ
ReaD
readln
READLN
ReadlN
write
WRITE
WritE
writeln
WRITELN
WritelN
