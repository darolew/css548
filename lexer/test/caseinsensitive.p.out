257   yand             
257   yand             
257   yand             
258   yarray           
258   yarray           
258   yarray           
260   ybegin           
260   ybegin           
260   ybegin           
262   ycase            
262   ycase            
262   ycase            
265   yconst           
265   yconst           
265   yconst           
267   ydiv             
267   ydiv             
267   ydiv             
269   ydo              
269   ydo              
269   ydo              
272   ydownto          
272   ydownto          
272   ydownto          
273   yelse            
273   yelse            
273   yelse            
274   yend             
274   yend             
274   yend             
277   yfor             
277   yfor             
277   yfor             
278   yfunction        
278   yfunction        
278   yfunction        
282   yif              
282   yif              
282   yif              
283   yin              
283   yin              
283   yin              
289   ymod             
289   ymod             
289   ymod             
292   ynil             
292   ynil             
292   ynil             
293   ynot             
293   ynot             
293   ynot             
296   yof              
296   yof              
296   yof              
297   yor              
297   yor              
297   yor              
299   yprocedure       
299   yprocedure       
299   yprocedure       
300   yprogram         
300   yprogram         
300   yprogram         
303   yrecord          
303   yrecord          
303   yrecord          
304   yrepeat          
304   yrepeat          
304   yrepeat          
308   yset             
308   yset             
308   yset             
310   ythen            
310   ythen            
310   ythen            
311   yto              
311   yto              
311   yto              
313   ytype            
313   ytype            
313   ytype            
314   yuntil           
314   yuntil           
314   yuntil           
315   yvar             
315   yvar             
315   yvar             
316   ywhile           
316   ywhile           
316   ywhile           
276   yfalse           
276   yfalse           
276   yfalse           
312   ytrue            
312   ytrue            
312   ytrue            
291   ynew             
291   ynew             
291   ynew             
266   ydispose         
266   ydispose         
266   ydispose         
301   yread            
301   yread            
301   yread            
302   yreadln          
302   yreadln          
302   yreadln          
317   ywrite           
317   ywrite           
317   ywrite           
318   ywriteln         
318   ywriteln         
318   ywriteln         
